features/dataset-view.feature

```diff
@@ -330,6 +330,45 @@ Feature: a user visit the dataset page
 		| pre_03AUG2015_update 								| folder containing originally submitted data files, prior to update Aug 3rd 2015.	| Directory 		| 50.00 MiB 	|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/pre_03AUG2015_update |
 		| readme.txt 										|				                                                                    | Readme 			| 337 B 		|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/readme.txt |
 
+	@ok @javascript @pr521
+	Scenario: To see the citation box and and the citation source
+		Given I am not logged in to Gigadb web site
+		And I am on "/dataset/101001"
+		And I should see a button input "Cite Dataset"
```

Comment:
@kencho51 can you check this test is working? I'm getting an error:
```
And I should see a button input "Cite Dataset"              # DatasetAdminContext::iShouldSeeAButtonInput()
      Failed asserting that false is true.
    │
    │  http://gigadb.dev/dataset/101001
```

Reply:
@pli888 I have updated the implementation to look for an `onclick` button, and the acceptance is passing now.